lib/src/button/Button.tsx

```diff
@@ -47,9 +47,7 @@ const DxcButton = ({
           disableRipple
           aria-disabled={disabled}
           tabIndex={disabled ? -1 : tabIndex}
-          onClick={() => {
```

Comment:
We need to undo this change. It was been called like that to prevent react from sending the `event` parameter to the user's `onClick` function. If we write `onClick={onClick}` and the user sends us a function like `(event) => {console.log(event);}` it will work and print the value, and we don't want that.

![image](https://user-images.githubusercontent.com/44321109/167364953-2b26abe0-0a46-46d0-92f0-a388adc0bd95.png)